packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/saml2/prep.md

```diff
@@ -1,12 +1,28 @@
-### Prepare a SAML integration
+If you haven't built the SAML 2.0 service in your app yet, review the [SAML concept](/docs/concepts/saml/) first.
 
-In a SAML integration, Okta is the Identity Provider (IdP), and your application is the Service Provider (SP). If you need more background on the protocol or for SAML best practices for your application, review our [SAML concept](/docs/concepts/saml/) documentation.
+### Use SAML toolkits
 
-Before you create a new SAML integration in Okta:
+The following open source and paid tool kits can help you implement SAML 2.0 for your existing app. These different language kits help you create the WebSSO Profile for Service Providers:
```

Comment:
again, do we want to say app integration? Optional

Reply:
I won't add "app integration" here because we are referring to the app, not the integration.